in_toto/in_toto_sign.py

```diff
@@ -151,8 +151,18 @@ def _sign_and_dump_metadata(metadata, args):
     # Alternatively we iterate over passed private key paths `--key KEYPATH ...`
     # load the corresponding key from disk and sign with it
     elif args.key != None: # pragma: no branch
-      for key_path in args.key:
-        key = util.prompt_import_rsa_key_from_file(key_path)
+
+      if args.key_type is None:
+        args.key_type = [util.KEY_TYPE_RSA] * len(args.key)
+
+      if len(args.key_type) != len(args.key):
+        raise securesystemslib.exceptions.FormatError(
+          "number of key_types should match with the number"
+          " of keys specified")
```

Comment:
The help description for --key-type states that "rsa" is used if one isn't given.  If more than one --key is given, is the user forced to specify all the key types?  I'm not sure that's clear from the help description.